README.md

```diff
@@ -40,10 +40,12 @@ class User(db.Model):
     username: Mapped[str] = mapped_column(unique=True)
 
 with app.app_context():
+    db.drop_all()
     db.create_all()
 
-    db.session.add(User(username="example"))
+    db.session.add(User(username="frank"))
     db.session.commit()
 
-    users = db.session.execute(db.select(User)).scalars()
+    users = db.session.scalars(db.select(User))
+    print({"users": [u.username for u in users]})
```

Comment:
I don't love over-encouraging print statements, but it is common in examples, so I see the reasoning.

I think this could at least be simplified, as the dict seems unneeded:

```suggestion
    print([u.username for u in users])
```

If you extend from MappedAsDataClass, then you can just print(users) but you don't get that nice repr with just the DeclarativeBase, so the list comprehension is warranted.

Reply:
Thanks for the review, but I see the dict a more friendly output here.